### ISSUE TITLE
nuget package update from 1.5.31 - DataTablesModelBinder

### ISSUE THREAD

**Issue by mcshaz** [mcintyre321/mvc.jquery.datatables#164]
I installed nuget package 1.5.31 (.net 4.5.1, NOT core), but the DataTablesParam was not binding the sSearchValues array. When I downloaded and included the relevant projects hosted on github, the binding worked perfectly.

This makes me suspect that at the time the nuget package was uploaded, there was an error within the DataTablesModelBinder code which has subsequently been corrected or reverted to a prior working version. If this is the case, could a new version be uploaded to nuget? Thank you.

As a small side note, the DataTablesModelBinder .BindModel method chooses between bind v10 and bind legacy - would you be able to add a comment to the code on what this refers to (exactly what version of what library) (I had assumed datatables 1.10.x but this does not seem to be the case).

Thank you

**Comment by mcintyre321** [comment]
Thanks for bringing this to my attention. I'm pushing 1.5.34 now, for some
reason I had disabled the nuget push from master, hope there wasn't a good
reason... I'll try and test it ASAP

**Comment by archana0909** [comment]
hi 
I installed nuget package 1.5.50 for .net(not core ) application but the DataTablesParam was not binding for any property at all while it is binding if I run the MvcJquery.DatatTable.Example project. I'm using DataTables 1.10.11 version. Should i be using a different version of datatables?
![s1](https://user-images.githubusercontent.com/7467518/77652741-c1790c00-6f94-11ea-80e9-21de9551a2eb.png)
![s2](https://user-images.githubusercontent.com/7467518/77652748-c342cf80-6f94-11ea-9895-d6eba2bfb18b.png)

**Comment by mcintyre321** [comment]
It looks like the model binder registration could be missing in your project. Check the files in app_start in your project

**Comment by archana0909** [comment]
yeah it seems the modelbinder class does not get downloaded with the nuget package. 
Added it manually and it worked. :) thanks
